inlang/source-code/sdk/src/loadProject.test.ts

```diff
@@ -52,7 +52,7 @@ const mockPlugin: Plugin = {
 	description: { en: "Mock plugin description" },
 	displayName: { en: "Mock Plugin" },
 
-	loadMessages: () => exampleMessages,
+	loadMessages: () => exampleMessages as Message[],
```

Comment:
the casting seems redundant. as typescript identifies the messages as `Message[]`


```suggestion
	loadMessages: () => exampleMessages,
```

Reply:
👍